src/main/java/gregtech/api/recipes/RecipeMap.java

```diff
@@ -417,7 +417,7 @@ protected void addInventorySlotGroup(ModularUI.Builder builder, IItemHandlerModi
         int startInputsY = 33 - (int) (itemSlotsToDown / 2.0 * 18) + yOffset;
         boolean wasGroup = itemHandler.getSlots() + fluidHandler.getTanks() == 12;
         if (wasGroup) startInputsY -= 9;
-        if (itemHandler.getSlots() == 6 && fluidHandler.getTanks() == 2 && !isOutputs) startInputsY -= 9;
+        else if (itemHandler.getSlots() >= 6 && fluidHandler.getTanks() >= 2 && !isOutputs) startInputsY -= 9;
```

Comment:
Why was this changed to else?

Reply:
Considering that there was already a case for when wasGroup was true, if I did not have an else, the yOffset would just go down twice (which would mess with the GUI). So, instead, I prevented this with an else.